ChatterTests/SocketTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SocketTests.swift
+//  ChatterTests
+//
+
+import XCTest
+@testable import Chatter
+
+class SocketTests: XCTestCase {
+
+    var socketService: SocketService!
+    var expectedMessage: Message!
+    var expectedChannel: Channel!
+    var correctMessageData: [Any]!
+    var corruptedMessageData: [Any]!
+    var correctChannelData: [Any]!
+    var corruptedChannelData: [Any]!
+    var emptyData: [Any]!
+
+    override func setUp() {
+        socketService = SocketService()
+        expectedMessage = Message(identifier: "identifier", messageBody: "messageBody", timeStamp: "timeStamp", channelId: "channelId", userName: "userName", userAvatar: "userAvatar", userAvatarColor: "userAvatarColor")
+        expectedChannel = Channel(identifier: "identifier", name: "name", description: "description")
+        correctMessageData = ["messageBody", "something", "channelId", "userName", "userAvatar", "userAvatarColor", "identifier", "timeStamp"]
+        corruptedMessageData = ["messageBody", "something", "channelId", "userName", "userAvatar", "userAvatarColor"]
+        correctChannelData = ["name", "description", "identifier"]
+        corruptedChannelData = ["name", "description"]
+        emptyData = []
+    }
+
+    func testCreateMessageFromCorrectData() {
+        guard let result = socketService.createMessageFrom(data: correctMessageData) else {
+            print("Error while unwrapping message from correct data")
```

Comment:
Use `XCTFail` to assure that this test will fail in case of problems with parsing.